public/app/plugins/panel/canvas/Connections.tsx

```diff
@@ -89,6 +90,15 @@ export class Connections {
   };
 
   handleMouseLeave = (event: React.MouseEvent | React.FocusEvent) => {
+    if (event.relatedTarget instanceof HTMLImageElement) {
+      const relatedTarget = event.relatedTarget;
+      assertInstanceOf(relatedTarget, HTMLImageElement);
```

Comment:
Is this line necessary?

Reply:
Yeah, otherwise cannot access relatedTarget.

Reply:
Good catch, much simpler now!